client/src/components/brushableHistogram/error.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import * as globals from "../../globals";
+
+const ErrorLoading = ({ displayName, error, zebra }) => {
+  console.log(error); // log to console as this is unexpected
```

Comment:
If this is intentionally trying to log an error to the console (as opposed to informational logging or debugging), then you should use `console.error()` rather than log().

Reply:
Also, we do not perform this action for any other type of error.  We should have a consistent policy and implement it everywhere.  Not just here.